src/alfasim_sdk/_internal/hook_specs.py

```diff
@@ -1785,6 +1785,8 @@ def calculate_relative_emulsion_viscosity(
     mu_disp: "double",
     mu_cont: "double",
     alpha_disp_in_layer: "double",
+    T: "double",
```

Comment:
I think that changing hook signatures will cause incompatibility with existing plugins.
And they will need to be updated (source code changed, not just recompiled).
Changelog entry about the breaking change?
Or we could go with creating an alternative version for the hook like `calculate_relative_emulsion_viscosity_per_temperature` or some think else in those lines (does `T` stands for temperature?).

I also think you should update the docstring to properly document the signature, I think that is what goes to the online documentation.

Reply:
This will break the existing plugins indeed and I forgot to update changelog and the docstring.

- [ ] Update changelog and docstring

> Or we could go with creating an alternative version for the hook like calculate_relative_emulsion_viscosity_per_temperature or some thing else in those lines (does T stands for temperature?

Yes, T is for temperature and creating another hook is not viable. Fortunately as far as I know there is nobody using this plugin, so this change probably won't bother anyone

Reply:
In fact I have updated the changelog. It was only the docstring that was missed

Reply:
> ... creating another hook is not viable ... there is nobody using this plugin...

OK... I will let it pass for now =P